Allows build with GCC 11. Based on changes in new p7zip version.

--- p7zip_16.02/CPP/7zip/Archive/Wim/WimHandler.cpp
+++ p7zip_16.02/CPP/7zip/Archive/Wim/WimHandler.cpp
@@ -298,7 +298,7 @@ }
 
       AString res;
 
-      bool numMethods = 0;
+      unsigned numMethods = 0;
       for (unsigned i = 0; i < ARRAY_SIZE(k_Methods); i++)
       {
         if (methodMask & ((UInt32)1 << i))
